cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -450,7 +451,16 @@ def test_update_from_json(self, store):
         altered_grader = CourseGradingModel.update_from_json(self.course.id, test_grader.__dict__, self.user)
         self.assertDictEqual(test_grader.__dict__, altered_grader.__dict__, "4 hour grace period")
 
-    def test_update_grader_from_json(self):
+        send_signal.assert_has_calls([
+            mock.call(sender=CourseGradingModel, user_id=self.user.id, course_id=self.course.id),
+            mock.call(sender=CourseGradingModel, user_id=self.user.id, course_id=self.course.id),
+            mock.call(sender=CourseGradingModel, user_id=self.user.id, course_id=self.course.id),
+            mock.call(sender=CourseGradingModel, user_id=self.user.id, course_id=self.course.id),
+            mock.call(sender=CourseGradingModel, user_id=self.user.id, course_id=self.course.id),
+        ])
```

Comment:
Because there's 5 calls to `update_from_json()`.  I can add a comment to this effect.